FMScoutFramework.sln

```diff
@@ -1,12 +1,18 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
+<<<<<<< HEAD
```

Comment:
It looks like you have commited an unresolved conflict. This will break the solution file. Please resolve before merging.